modules/itemManager.ts

```diff
@@ -53,6 +53,23 @@ class ItemManager {
   private sortItemsByPurchaseStatus = (items: Item[]): Item[] => {
     return items.sort((a: any, b: any) => a.isPurchased - b.isPurchased);
   };
+  private async countItemsInSection(
+    isOtherSection: boolean,
+    id: number
+  ): Promise<number> {
+    const item = await db.items.get(id);
```

Comment:
I'm still doing this in the `removeItem` method as well... it could be improved. But I want to move on to getting other parts more polished and adding `Lists` - I suspect that's going to require parts of this method to change (having to source the list and the item first, for example), so we'll have time to refactor and improve then.